lib/iris/analysis/maths.py

```diff
@@ -585,16 +575,17 @@ def _binary_op_common(operation_function, operation_name, cube, other,
                            `cube` and `cube.data`
     """
     _assert_is_cube(cube)
-
     if isinstance(other, iris.coords.Coord):
         other = _broadcast_cube_coord_data(cube, other, operation_name, dim)
     elif isinstance(other, iris.cube.Cube):
         try:
-            BA.broadcast_arrays(cube._my_data, other._my_data)
+            da.core.broadcast_shapes(cube.shape, other.shape)
```

Comment:
This can just be 'broadcast_shapes' rather than 'da.core.broadcast_shapes', right?